src/updi_state.h

```diff
@@ -37,9 +37,10 @@ typedef enum
 
 typedef enum
 {
-  UPDI_NVM_MODE_V0,
-  UPDI_NVM_MODE_V2,
-  UPDI_NVM_MODE_V3
+  UPDI_NVM_MODE_V0,   // tinyAVR-0/1/2 and megaAVR-0 series
+  UPDI_NVM_MODE_V2,   // AVR_DA/DB/DD series
+  UPDI_NVM_MODE_V3,   // AVR_EA series
+  UPDI_NVM_MODE_V5    // AVR_EB series (NEW)
```

Comment:
Up until this point - I wouldn't introduce new enum value, if the point is not to use it anyway.

Reply:
this will revert